pre_processing/rttov_driver.F90

```diff
@@ -929,7 +932,7 @@ subroutine rttov_driver(coef_path, emiss_path, granule, preproc_dims, &
          call rttov_deallocate_emis_atlas(emis_atlas)
          call rttov_alloc_traj(stat, 1, nchan, opts, nlevels, coefs, DEALLOC, &
               traj)
-         call rttov_alloc_transmission(stat, transmission, nlevels, nevals, &
+         call rttov_alloc_transmission(stat, transmission, nlevels, nchan, &
               DEALLOC)
          call rttov_alloc_rad(stat, nevals, radiance, nlevels, DEALLOC, &
```

Comment:
As best I can tell, `nevals` appears three times in this code: (1) in your comment, (2) when it's declared, and (3) in `rttov_alloc_rad`. I can't see it being assigned a value, so isn't it also wrong in that call?

Reply:
You are, as usual, correct. I've updated accordingly - nevals is no more.